### ISSUE TITLE
Новый клиент

### ISSUE THREAD

**Issue by GlendaleBrown** [AvaCity/avacity-test#4]
Нужен новый или почти новый клиент. Игра перестала запускаться при апдейте

**Comment by GlendaleBrown** [comment]
Напишите туториал

**Comment by exstrim401** [comment]
только когда сервер будет более менее работоспособен

**Comment by exstrim401** [comment]
Вкратце нужно заменить ip игрового и account сервера в xml файле внутри pnz-city, ссылку на вк поменять на свою и всё, попозже напишу статью

**Comment by Noop1** [comment]
"Игра перестала запускаться при апдейте", видимо у вас не грузятся файлы boyClothes.xml / girlClothes.xml, как не странно, в обычном конфиге его нет, вот и проблема.

**Comment by exstrim401** [comment]
> "Игра перестала запускаться при апдейте", видимо у вас не грузятся файлы boyClothes.xml / girlClothes.xml, как не странно, в обычном конфиге его нет, вот и проблема.

Пост с 19 апреля, в то время boyClothes и girlClothes были в конфиге

**Comment by Noop1** [comment]
Но использовав 2.0 версию в то время, в апдейте всё было нормально, странно.

**Comment by Noop1** [comment]
@exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)

**Comment by exstrim401** [comment]
> @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)

Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю

**Comment by SweetLotusMoon** [comment]
> > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> 
> Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю

уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...

**Comment by JayweTee** [comment]
> > > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> > 
> > 
> > Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю
> 
> уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...

Чо ты несёшь поехавший 😦

**Comment by SweetLotusMoon** [comment]
> > > > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> > > 
> > > 
> > > Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю
> > 
> > 
> > уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...
> 
> Чо ты несёшь поехавший 😦

тогда, по твоему мнению, как клиенты с аватарии парсили бы id одежды, их стоимость...?

**Comment by SweetLotusMoon** [comment]
> > > > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> > > 
> > > 
> > > Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю
> > 
> > 
> > уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...
> 
> Чо ты несёшь поехавший 😦

К тому же, не стоило так открыто меня оскорблять, я вам даже ещё ничего не сделал

**Comment by JayweTee** [comment]
> > > > > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> > > > 
> > > > 
> > > > Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю
> > > 
> > > 
> > > уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...
> > 
> > 
> > Чо ты несёшь поехавший 😦
> 
> тогда, по твоему мнению, как клиенты с аватарии парсили бы id одежды, их стоимость...?

Тогда конфиг перенесли с клиента на сервер. Сервер отдавал по запросу клиента ответ в котором находилась вся информация об одежде. Как это работает сейчас я не знаю, да и не интересно это уже.

**Comment by SweetLotusMoon** [comment]
> > > > > > @exstrim401, не знаю создавать ли для этого отдельный пост, но откуда брать тогда файлы одежды? (boyClothes, girlClothes)
> > > > > 
> > > > > 
> > > > > Пока ниоткуда, использовать старый клиент. Возможно потом что-нибудь придумаю
> > > > 
> > > > 
> > > > уже появилось, к тому же в то время можно было просто найти то содержимое где находятся данные об одежде в клиенте аватарии, либо дописав...
> > > 
> > > 
> > > Чо ты несёшь поехавший 😦
> > 
> > 
> > тогда, по твоему мнению, как клиенты с аватарии парсили бы id одежды, их стоимость...?
> 
> Тогда конфиг перенесли с клиента на сервер. Сервер отдавал по запросу клиента ответ в котором находилась вся информация об одежде. Как это работает сейчас я не знаю, да и не интересно это уже.

Да, я о этом и имел речь, но по другому сказал(содержимое, имел ввиду код), то есть в клиенте есть алгоритм, который принимает и отдает команды серверу об одежде, извините.
